x-pack/metricbeat/module/aws/cloudwatch/metadata/ec2/ec2.go

```diff
@@ -159,20 +163,15 @@ func addHostFields(event mb.Event, instanceID string) {
 		}
 
 		if value, ok := metricValue.(float64); ok {
-			if ec2MetricName == "cpu.total.pct" {
+			if ec2MetricName == "aws.ec2.metrics.CPUUtilization.avg" {
 				value = value / 100
 			}
 			_, _ = event.RootFields.Put(hostMetricName, value)
 		}
 	}
 }
 
-func calculateRate(event mb.Event, monitoringState string) {
-	var period = 300.0
-	if monitoringState != "disabled" {
-		period = 60.0
-	}
-
+func calculateRate(event mb.Event, period int) {
```

Comment:
nit: 

please can we either, change the name of the arg to `periodInSeconds` or something similar, or require it as a `time.Duration`?